roles/openshift_health_checker/library/aos_version.py

```diff
@@ -32,10 +33,14 @@
     PKG_MGR = "yum"
 except ImportError as err:
     YUM_IMPORT_EXCEPTION = err
+
 try:
+    # We need to check whether the dnf binary is installed, since the
+    # import of dnf module won't fail even if the dnf binary is missing
+    subprocess.call(["dnf"])
```

Comment:
Both [ansible/modules/packaging/os/yum.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/packaging/os/yum.py) and [ansible/modules/packaging/os/dnf.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/packaging/os/dnf.py) use the Python libraries to interact with the package manager, and not their command line binaries.